resources/views/sports/create.blade.php

```diff
@@ -13,14 +13,14 @@
                                 @csrf
 
                                 <div class="form-group row">
-                                    <label for="first" class="col-md-4 col-form-label text-md-right">1st place:</label>
+                                    <label for="1-{{ $sport->id }}" class="col-md-4 col-form-label text-md-right">1st place:</label>
 
                                     <div class="col-md-6">
-                                        <select name="first" id="first"
+                                        <select name="1-{{ $sport->id }}" id="1-{{ $sport->id }}"
```

Comment:
@Evolyuta not sure that doing it with X-name is the best solution, it may be better to use arrays. Will talk about it in the video.